src/templates/docker-templates/jellyfin.yaml

```diff
@@ -35,4 +35,5 @@ jellyfin:
     - ./jellyfin/data/misc/metadata:/data/metadata
     - ./jellyfin/data/misc/cache:/data/cache
     - ./jellyfin/data/misc/databases:/config/data/data
+    - ./media:/data/media

```

Comment:
Should replace `/data/movies` and `/data/tvshows` defined 5 lines up?

Reply:
Yes.